react/TranslateEstimate.test.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+
```

Comment:
(related to the whole file)
tests were failing?

Reply:
yep, we changed the translations but didn't update the tests accordingly